### PR TITLE
Remove unused bundles

### PR DESCRIPTION
**Pull request by gmrukwa**
There was a problem with bundled version of API - JS files were absent, as they are currently unused. It also appears that API help does not work properly yet, but it is scheduled as a separate task now.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[Fix bundling of JS missing in API package](https://trello.com/c/4rX5irhp/212-fix-bundling-of-js-missing-in-api-package)

File: src/Spectre/Scripts/respond.js [hunk 1]
```diff
@@ -1,340 +0,0 @@
-/* NUGET: BEGIN LICENSE TEXT
```
Comment (Sand3r-) [inline src/Spectre/Scripts/respond.js]: Isn't the file left empty?

Comment (Sand3r-) [inline src/Spectre/Scripts/respond.js]: And if so, does it need to exist?

Comment (Regis128) [inline src/Spectre/Scripts/respond.js]: I think so too, better to delete the file if not necessery

Comment (mg6) [inline src/Spectre/Scripts/respond.js]:
```
% git diff --name-status develop origin/fixup/remove-unused-bundles
M   appveyor.yml
M   src/Spectre/App_Start/BundleConfig.cs
D   src/Spectre/Scripts/_references.js
D   src/Spectre/Scripts/bootstrap.js
D   src/Spectre/Scripts/bootstrap.min.js
D   src/Spectre/Scripts/jquery-1.10.2.intellisense.js
D   src/Spectre/Scripts/jquery-1.10.2.js
D   src/Spectre/Scripts/jquery-1.10.2.min.js
D   src/Spectre/Scripts/jquery-1.10.2.min.map
D   src/Spectre/Scripts/jquery.validate-vsdoc.js
D   src/Spectre/Scripts/jquery.validate.js
D   src/Spectre/Scripts/jquery.validate.min.js
D   src/Spectre/Scripts/jquery.validate.unobtrusive.js
D   src/Spectre/Scripts/jquery.validate.unobtrusive.min.js
D   src/Spectre/Scripts/modernizr-2.6.2.js
D   src/Spectre/Scripts/respond.js
D   src/Spectre/Scripts/respond.min.js
```
Seems fine. Those files are completely deleted.

Comment (gmrukwa) [inline src/Spectre/Scripts/respond.js]: @Sand3r-, @Regis128, I confirm. These files are completely deleted.

**Pull request merged**